ios/FluentUI/Button/Button.swift

```diff
@@ -241,7 +241,7 @@ open class Button: UIButton {
     }
 
     private func updateImage() {
-        let isDisplayingImage = (style == .primaryFilled || style == .primaryOutline || style == .secondaryOutline) && image != nil
+        let isDisplayingImage = (style == .primaryFilled || style == .primaryOutline || style == .secondaryOutline || style == .borderless) && image != nil
```

Comment:
nit : with this long list i wonder it will be more simplified if we check NOT .tertiaray.